actions.yaml

```diff
@@ -1,12 +1,6 @@
 #!/usr/bin/env python3
-# Copyright 2021 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-authorise-charm:
-  description: Authorise the Vault charm to interact with Vault.
-  properties:
-    token:
-      type: string
-      description: Token to use to authorise charm.
-  required:
-  - token
+get-root-token:
+  description: Returns Vault's root token.
```

Comment:
Isn't this token a "secret"?

Reply:
Yes it is

Reply:
This made me think that users can now access secrets created by charms so I removed this action completely and pointed users to using the `juju secret` cli endpoint